canonical/maas-ui
# test: update sitespeed scripts

## Done

- test: update sitespeed scripts
- update wait.byCondition selector
- update sitespeed script to be compatible with all platforms (linux, mac, windows)
- rename to sitespeed:ci

## QA

### QA steps

- Ensure you have docker running
- Run `yarn sitespeed`
- Test should complete with no errors and results should be added to `sitespeed.io/results`
